### ISSUE TITLE
Timeout of undefineds exceeded!

### ISSUE THREAD

**Issue by andreinanu** [SeydX/homebridge-bravia-tvos#110]
The plugin was working fine until a couple of days ago, when suddenly `Bravia: Timeout of undefineds exceeded!` shows up in the logs. I also see this polling every few seconds:

`[29/11/2020, 17:15:05] [Bravia] Bravia: {"uri":"extInput:hdmi?port=2","source":"extInput:hdmi","title":"HDMI 2"}
[29/11/2020, 17:15:15] [Bravia] Bravia: Polling tv state
[29/11/2020, 17:15:21] [Bravia] Bravia: {"uri":"extInput:hdmi?port=2","source":"extInput:hdmi","title":"HDMI 2"}
[29/11/2020, 17:15:31] [Bravia] Bravia: Polling tv state
[29/11/2020, 17:15:37] [Bravia] Bravia: {"uri":"extInput:hdmi?port=2","source":"extInput:hdmi","title":"HDMI 2"}`

Since this started, the TV sets to non-responsive every time I try to switch it on/off, with the following feedback:

`[29/11/2020, 17:17:03] [Bravia] Bravia: Timeout of undefineds exceeded!
[29/11/2020, 17:17:13] [Bravia] Bravia: Polling tv state
[29/11/2020, 17:17:19] [Bravia] Bravia: An error occured during powering off tv!
[29/11/2020, 17:17:19] [Bravia] Bravia: Timeout of undefineds exceeded!
[29/11/2020, 17:17:21] [Bravia] Bravia: {"uri":"extInput:hdmi?port=2","source":"extInput:hdmi","title":"HDMI 2"}`

Other plugins currently installed:
Homebridge Weather
Homebridge Airvisual 2
Homebridge Samsung Tizen
Homebridge Tplink Smarthome
Homebridge Tplink Smarthome
Homebridge UI

This is running on the latest version of Homebridge on a RPI 4b.

Any idea what can be wrong? 
In case I can provide anything else, please let me know.

Thanks!

**Comment by andreinanu** [comment]
@SeydX I've reinstalled the plugin and the setup process was seamless (amazing job, btw!), I have no errors in HB but seems that I cannot add it to Home app - restarted the PI, TV and phone and still cannot discover it. I've also tried adding the code manually but to no avail. I guess I can try to remove the hub completely from Home and rescan it - if you have any other suggestion, it's very welcomed.
